modules/deployment-model/src/main/java/org/mule/runtime/deployment/model/api/application/ApplicationPolicyManager.java

```diff
@@ -16,7 +16,7 @@
 public interface ApplicationPolicyManager {
 
   /**
-   * Add a new policy
+   * Adds and initialises a new policy
```

Comment:
No need to update this javadoc regarding the initialization. But it should return a boolean indicating if it was added successfully or not, throwing an exception is not good